LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -940,10 +940,10 @@ TacticalText="<Bullet/> Coil weapons automatically pierce (ignore) 1 enemy armor
 AbilityDescName="sniper rifle"
 
 [Shotgun_CG X2WeaponTemplate]
-FriendlyName="Scattergun"
-FriendlyNamePlural="Scatterguns"
+FriendlyName="Gauss Scattergun"
+FriendlyNamePlural="Gauss Scatterguns"
```

Comment:
Please surround modified localisation text with:
```
; LWOTC Needs Translation
...
; End Translation
```
Only put those comment lines around text that is new or has changed, so _not_ around the `BriefSummary=` line.